docs/terms/metric.md

```diff
@@ -4,12 +4,104 @@ title: Metric
 hoverText: A computed attribute of data such as the mean of a column.
 ---
 import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import TechnicalTag from '../term_tags/_tag.mdx';
+import CreateHeader from '/docs/images/universal_map/_um_create_header.mdx';
+import ValidateHeader from '/docs/images/universal_map/_um_validate_header.mdx';
 
-<UniversalMap setup='active' connect='active' create='active' validate='active'/>
+
+<UniversalMap setup='inactive' connect='inactive' create='active' validate='active'/> 
+
+## Overview
+
+### Definition
 
 A Metric is a computed attribute of data such as the mean of a column.
 
+### Features and promises
+
+Metrics are values derived from one or more <TechnicalTag relative="../" tag="batch" text="Batches" /> that can be used to evaluate <TechnicalTag relative="../" tag="expectation" text="Expectations" /> or to summarize the result of <TechnicalTag relative="../" tag="validation" text="Validation" />. A Metric could be a statistic, such as the minimum value of the column, or a more complex object, such as a histogram. Metrics are the core tool used to Validate data. 
+
+### Relationship to other objects
+
+Metrics are generated and stored as part of running Expectations against a Batch (and can be referenced as such). For example, if you have an Expectation that the mean of a column fall within a certain range, the mean of the column must first be computed to see if its value is as expected.  The generation of Metrics involves <TechnicalTag relative="../" tag="execution_engine" text="Execution Engine" /> specific logic.  Metrics from previously run Expectation Suites can also be referenced through the use of <TechnicalTag relative="../" tag="evaluation_parameter" text="Evaluation Parameters" />.
```

Comment:
It might be ambiguous/confusing to say that Metrics are stored. They are stored in memory so they can be used in the current validation. They can also be stored within ExpectationValidationResults, though not all computed metrics will be, and this will also depend on `result_format`.

They _can_ be stored in a Metric Store, though this functionality is still pretty experimental and not totally recommend (though design work for an upgrade is in-progress).

```suggestion
Metrics are generated and stored as part of running Expectations against a Batch (and can be referenced as such). For example, if you have an Expectation that the mean of a column falls within a certain range, the mean of the column must first be computed to see if its value is as expected.  The generation of Metrics involves <TechnicalTag relative="../" tag="execution_engine" text="Execution Engine" /> specific logic.  Metrics from previously run Expectation Suites can also be referenced through the use of <TechnicalTag relative="../" tag="evaluation_parameter" text="Evaluation Parameters" />.
```

Reply:
I made this edit and clarified how some metrics, based on `result_format` of Validation Results, can be accessed through in memory Validation Results by Actions, which can then store them so that they can be referenced from the Validation Results Store later.